datadog/resource_datadog_synthetics_global_variable.go

```diff
@@ -48,7 +48,7 @@ func resourceDatadogSyntheticsGlobalVariable() *schema.Resource {
 				Sensitive:   true,
 			},
 			"secure": {
-				Description: "Sets the variable as secure. Defaults to `false`.",
+				Description: "If set to true, the value of the global variable will be hidden. Defaults to `false`.",
```

Comment:
Going to need to run `make docs`
Also I think 
```suggestion
				Description: "If set to true, the value of the global variable is hidden. Defaults to `false`.",
```

Reply:
+1 on that, we try to avoid future tense as often as we can!